src/simple_tracker_interfaces/msg/ObserverCloudEstimation.msg

```diff
@@ -0,0 +1,4 @@
+# Defines the message for the environmental data supplied by the station
+
+# The percentage cloud cover currently being determined
+float32 percentage_cloud_cover
```

Comment:
In practice, how will this value be used?

Reply:
The idea is that some bgs algos might be better suited to cloudy periods than others or we might need to tweaks parameters slightly, so it's to help with that.

Reply:
It might also come in handy further down the line with tracking and the PTF.  Was thinking as well that we could possibly look to estimate cloud height - might need two cams for that though... not sure